yarn.lock

```diff
@@ -2,12 +2,12 @@
 # yarn lockfile v1
 
 
-"@nosto/nosto-react@^0.4.1":
-  version "0.4.1"
-  resolved "https://registry.yarnpkg.com/@nosto/nosto-react/-/nosto-react-0.4.1.tgz#2c497d0df4d0c961ee6166d18cebbf70df46385d"
-  integrity sha512-vNlOCf1IZDX2gNUz4vcGP040Leyazqw6jNSiqY2txiuB/jMnz9f3ruDEPAiY6gjx3uOaItZFsDPoXUbffGCDtg==
+"@nosto/nosto-react@^2.2.2":
+  version "2.2.2"
+  resolved "https://npm.nos.to/@nosto%2fnosto-react/-/nosto-react-2.2.2.tgz#bd70dba325abc6126b9b7bc4ac79b6aec6e25d1f"
```

Comment:
use a public repo instead

Reply:
FYI, This could happen if the repo is configured using `npm config set registry` or if the intel Mac is used